app/services/container_topology_service.rb

```diff
@@ -79,7 +79,7 @@ def entity_status(entity)
     elsif entity.kind_of?(ContainerReplicator)
       status = (entity.current_replicas == entity.replicas) ? 'OK' : 'Warning'
     elsif entity.kind_of?(ManageIQ::Providers::ContainerManager)
-      status = entity.authentications.empty? ? 'Unknown' : entity.authentications.first.status.capitalize
+      status = entity.authentications.empty? ? 'Unknown' : entity.authentications.find_by_authtype('bearer').status.capitalize
```

Comment:
@zeari I don't think this authtype selection logic belongs to the topology. I would prefer to see this being moved closer to the authentication mixin.